CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Changelog
 
+## 11.0.0
```

Comment:
Note: @russjones has a separate PR open with breaking changes to the changelog for v11. can we move this over to that branch?

Reply:
Should I just move the changelog changes to that branch or change the base of this PR?

Reply:
Went ahead and moved changelog changes to changelog PR.